### PR TITLE
Check leaves for changes before dumping

### PR DESCRIPTION
**Pull request by artursvonda**

### CODE REVIEW

**Comment by robertfausk** [comment]
I think this is the follwer of https://github.com/symfony/AsseticBundle/pull/233

Will there be any progress on this?

**Comment by artursvonda** [comment]
I rebased my PR. There are some failures in tests, but looks like they are because dev-master version of some of the symfony components where used (meaning, Symfony 3.0 components). Not sure, where the bug is, but it's not bundle's fault. Everything works if dependencies are installed with '--prefer-stable'.

**Comment by xabbuh** [comment]
@artursvonda Can't you simply remove the `minimum-stability` setting from the `composer.json` file?

**Comment by robertfausk** [comment]
@artursvonda ping:)

**Comment by artursvonda** [comment]
The issue is not on my end and I'm not sure if the issue is with Symfony component versioning or composer. The composer.json file looks correct.

File: Command/DumpCommand.php [hunk 1]
```diff
@@ -110,12 +111,12 @@ protected function execute(InputInterface $input, OutputInterface $stdout)
             );
 
             $self = $this;
-            $batch->execute(function ($name) use ($self, $stdout) {
-                $self->dumpAsset($name, $stdout);
+            $batch->execute(function($name) use($self, $stdout, &$previous) {
```
Comment (stof) [inline Command/DumpCommand.php]: will the reference really work when forking the process ?

File: Command/DumpCommand.php [hunk 2]
```diff
@@ -110,12 +111,12 @@ protected function execute(InputInterface $input, OutputInterface $stdout)
             );
 
             $self = $this;
-            $batch->execute(function ($name) use ($self, $stdout) {
```
Comment (stof) [inline Command/DumpCommand.php]: spaces should not be removed

**Pull request closed without merging**